compute_tools/src/compute.rs

```diff
@@ -958,11 +958,15 @@ impl ComputeNode {
         let mut delay_exit = false;
         let mut state = self.state.lock().unwrap();
         state.terminate_flush_lsn = lsn;
-        if let ComputeStatus::TerminationPending { mode } = state.status {
+        if state.status == ComputeStatus::TerminationPendingFast {
+            state.status = ComputeStatus::Terminated;
+            self.state_changed.notify_all();
+            delay_exit = true
+        } else if state.status == ComputeStatus::TerminationPendingImmediate {
             state.status = ComputeStatus::Terminated;
             self.state_changed.notify_all();
             // we were asked to terminate gracefully, don't exit to avoid restart
-            delay_exit = mode == compute_api::responses::TerminateMode::Fast
+            delay_exit = false
```

Comment:
Could you please add logs related to status change like what `ComputeNode::set_status()` does?

```
        info!("Changing compute status from {} to {}", prev, status);
```

Reply:
> No, we don't want to update COMPUTE_CTL_UP

Why do we not want to do this?

Reply:
I thought this could do weird things to activity monitor, on second thought, I found out it doesn't use _UP metric, so fine to go this way

Reply:
Because we have some dependency on that in tests, so arm tests fail, and I don't want to investigate this in this task